elytron/WFCORE-4362-configurable-certificate-authority-in-certificate-authority-account.adoc

```diff
@@ -0,0 +1,66 @@
+= Make the certificate authority used by a certificate-authority-account configurable
+:author:            Farah Juma, Diana Vilkolakova
+:email:             dvilkola@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Currently, Let's Encrypt (https://acme-v02.api.letsencrypt.org/directory) is the only certificate authority
+that can be used in a `certificate-authority-account` resource.
+This issue aims to make it possible for the user to configure attribute `certificate-authority` for this resource.
+Each certificate authority can be associated with a production url and a staging url, both of which have to be configurable.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFCORE-4362[WFCORE-4362]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1150
+
+=== Dev Contacts
+
+* mailto:dvilkola@redhat.com[Diana Vilkolakova]
+
+=== QE Contacts
+
+* mailto:mchoma@redhat.com[Martin Choma]
+
+=== Affected Projects or Components
+
+* Wildfly Core
+* Elytron
+
+=== Other Interested Projects
+
+== Requirements
+
+=== Hard Requirements
+
+It must be possible to specify staging url and production url for the attribute `certificate-authority` in the `certificate-authority-account` resource.
+
```

Comment:
I think it would be good to give an example here of what the configuration would look like. (It's described well in the next section but often helps to be able to see the configuration as well.)